src/app/components/Modals/Terms/data/terms.tsx

```diff
@@ -1,798 +1,1012 @@
+        Provider cannot anticipate, control or influence the occurrence or
+        outcome of Forks, and do not assume any risk, liability or obligation in
+        connection therewith. Provider does not assume any responsibility to
+        notify you of pending, threatened or completed Forks. Provider will
+        respond (or refrain from responding) to any Forks in such manner as
+        Provider determines in its sole and absolute discretion. Provider will
+        not have any duty or obligation, or liability to any user if such
+        response (or lack of such response) acts to your detriment. You assume
+        full responsibility to independently remain apprised of and informed
+        about possible Forks, and to manage your own interests and risks in
+        connection therewith.
       </p>
       <br />
-      <h5 className="text-primary">11.2 No Regulatory Supervision</h5>
+      <h3>9. TAX ISSUES</h3>
       <br />
       <p>
-        The Interface maintainers and the Interface are not registered or
-        qualified with or licensed by, do not report to, and are not under the
-        active supervision of any government agency or financial regulatory
-        authority or organization. No government or regulator has approved or
-        has been consulted by the Interface maintainers regarding the accuracy
-        or completeness of any information available on the Interface.
-        Similarly, the technology, systems, blockchains, tokens, and persons
-        relevant to information published on the Interface may not be registered
-        with or under the supervision of or be registered or qualified with or
-        licensed by any government agency or financial regulatory authority or
-        organization. The Interface maintainers are not registered as brokers,
-        dealers, advisors, transfer agents or other intermediaries.
+        The tax consequences of purchasing, selling, holding, transferring, or
+        locking Digital Assets or otherwise utilizing the Services are uncertain
+        and may vary by jurisdiction. Provider has undertaken no due diligence
+        or investigation into such tax consequences, and you assume all
+        obligation and liability for the tax consequences applicable to you.
       </p>
       <br />
-      <h5 className="text-primary">11.3 Regulatory Uncertainty</h5>
+      <h3>10. LIMITATION OF LIABILITY</h3>
       <br />
       <p>
-        Blockchain technologies and digital assets are subject to many legal and
-        regulatory uncertainties, and the Protocol or any tokens or blockchains
-        could be adversely impacted by one or more regulatory or legal
-        inquiries, actions, suits, investigations, claims, fines, or judgments,
-        which could impede or limit the ability of User to continue the use and
-        enjoyment of such assets and technologies.
+        Except to the extent otherwise required by applicable law, under no
```

Comment:
This paragraph should be bold as in the document

Reply:
Done